Recently released Lua 5.4 introduced additional argument in lua_resume function.

Taken from upstream:
https://github.com/pavouk/lgi/pull/199

--- lgi-0.9.2/lgi/callable.c
+++ lgi-0.9.2/lgi/callable.c
@@ -1233,7 +1233,10 @@ }
     }
   else
     {
-#if LUA_VERSION_NUM >= 502
+#if LUA_VERSION_NUM >= 504
+      int nresults;
+      res = lua_resume (L, NULL, npos, &nresults);
+#elif LUA_VERSION_NUM >= 502
       res = lua_resume (L, NULL, npos);
 #else
       res = lua_resume (L, npos);
